implementations/rust/ockam/ockam_command/src/lib.rs

```diff
@@ -310,6 +310,8 @@ pub fn run() {
 
 impl OckamCommand {
     pub fn run(self) {
+        let _hook_result = miette::set_hook(Box::new(|_| Box::new(ErrorReportHandler::new())));
```

Comment:
Can we have a small comment here explaining why we need this line of code and what are the consequences?